packages/client/modules/teamDashboard/components/TeamSettingsWrapper/TeamSettingsWrapper.tsx

```diff
@@ -23,17 +22,12 @@ const IntegrationPage = styled('div')({
   flexDirection: 'column'
 })
 const TeamSettingsWrapper = (props: Props) => {
-  const {
-    location: {pathname},
-    match
-  } = props
+  const {match} = props
   const {
     params: {teamId}
   } = match
-  const areaMatch = matchPath(pathname, {path: `${match.url}/:area?`}) || {params: {area: ''}}
   return (
     <IntegrationPage>
-      <TeamSettingsToggleNav activeKey={(areaMatch.params as any).area || ''} teamId={teamId} />
       <Switch>
```

Comment:
-1 We don't need this `<Switch>` here anymore. In fact, we can remove the whole file and just use the `TeamSettingsRoot` component in the place where we previously used this `TeamSettingsWrapper` component.